fpmsyncd/fpmsyncd.cpp

```diff
@@ -122,17 +136,26 @@ int main(int argc, char **argv)
                  * select() loop.
                  * Note:  route reconciliation always succeeds, it will not be done twice.
                  */
-                if (temps == &warmStartTimer || temps == &eoiuHoldTimer)
+                if (temps == &warmStartTimer || temps == &eoiuHoldTimer || temps == &reconcileHoldTimer)
                 {
+                    bool readyToReconcile = false;                    
                     if (temps == &warmStartTimer)
                     {
+                        readyToReconcile = true;
```

Comment:
In case of orchagent still not ready, should we just abort?

Reply:
reconcile should take care of cleaning up stale entries. So not sure abort is necessary.

Reply:
there was a reason we wait for "sync.isReadyToReconcile", I assume it was a must condition for reconcile to be working as expected.   Again, if that condition is broken, we should make it visible to user.  This probably won't happen in normal case, if it does, we should have information for user to debug, so raise or abort could help.

Reply:
again same as earlier, orchagent will eventually reconcile. If it does not, it would have its own mechanism to recover/abort. Here we are making effort to reconcile the system to pre-warm-reboot state. Hence if we continue, we would reconcile prematurely and hence warm-reboot may not be hitless. However if we abort, we would impact everything and the full traffic will be hit.